cartridges/openshift-origin-cartridge-python/usr/versions/2.7-scl/lib/update-configuration

```diff
@@ -8,19 +8,20 @@ function update-configuration {
     # SCL installs of python 2.7
     sclpath=$(dirname $(scl enable python27 "which python"))
     env_dir="${OPENSHIFT_PYTHON_DIR}/env"
```

Comment:
@abhgupta `${env_dir}` is not used in the value of any environment variables, but only to point to the location of the environment variable. Double slashes should be acceptable as long as they aren't in the value of the env variable.